core/example/benchmark/CMakeLists.txt

```diff
@@ -3,5 +3,7 @@ if(Cabana_ENABLE_Serial)
   target_link_libraries(NeighborListMDPerfTest cabanacore)
 endif()
 
-add_executable(PeakFlops Cabana_peakflops.cpp)
-target_link_libraries(PeakFlops cabanacore)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
```

Comment:
Did you mean to have x86 OR amd?  Your PR subject says "only on x86"

Reply:
Why do you have the patterns between parentheses?
Why do you provide uppercase variants?